algosdk/atomic_transaction_composer.py

```diff
@@ -255,15 +256,41 @@ def __init__(
         decode_error: Optional[Exception],
         tx_info: dict,
         method: abi.Method,
-        missing_signature: bool,
     ) -> None:
         self.tx_id = tx_id
         self.raw_value = raw_value
         self.return_value = return_value
         self.decode_error = decode_error
         self.tx_info = tx_info
         self.method = method
-        self.missing_signature = missing_signature
+
+
+class SimulateEvalOverrides:
+    def __init__(
+        self,
+        *,
+        max_log_calls: Optional[int] = None,
```

Comment:
Should we address `allow-empty-signatures` in another PR?

Reply:
good callout, I will put the field here soon

Reply:
see [pr response](https://github.com/algorand/py-algorand-sdk/pull/469/commits/939e1216a567006d6667ecd622965788719a41cf)